cypress/support/test-utils.ts

```diff
@@ -4,17 +4,23 @@ import type { PathConfig } from '../../src/app/_constants/paths'
 import { verifyCanonicalLink } from '../support/verifyCanonicalLinkUtil'
 import { verifyPageTitle } from '../support/verifyPageTitleUtil'
 
-export function testPageMetadata({
-  path: { mainContentPath, path, entriesContentPath },
-  hasPageHeaderDescription = true,
-  includesFeaturedEntry = false,
-  overrideDefaultTitle = false,
-}: {
-  path: PathConfig
+type TestMetaDataOptions = {
   hasPageHeaderDescription?: boolean
   includesFeaturedEntry?: boolean
   overrideDefaultTitle?: boolean
-}) {
+}
+
+export function testPageMetadata(
```

Comment:
line 33 we should add  `header: { title: string; description: string | string[] }`

Reply:
Resolved live with @barbaraperic